### ISSUE TITLE
Support on-premise/self-hosted Jira

### ISSUE THREAD

**Issue by SimoonTech** [ajgassner/jira-pretty-board#13]
Hi @ajgassner 

Great peace of software, I've however hit a small wall, see below:

<img width="1680" alt="Screen Shot 2019-09-02 at 22 47 29" src="https://user-images.githubusercontent.com/23457220/64134202-c184a500-cdd3-11e9-8d99-389a92c9329a.png">

my credentials are correct -- is there some form of debugging option?

**Comment by ajgassner** [comment]
Hi,

you need to have active sprints configured in your Jira Cloud instance. You can debug e.g. with Chrome developer tools.

**Comment by SimoonTech** [comment]
HI @ajgassner 
thanks for getting back to me on this, 
we have an on perm Jira Server -- i guess this will not work for what we have here.
i'll keep digging around

**Comment by ajgassner** [comment]
@SuperDuperOne yes, self-hosted Jira is currently not supported. Fell free to raise a pull request.
